ext/pspell/pspell.stub.php

```diff
@@ -4,38 +4,45 @@
 
 function pspell_new(string $language, string $spelling = "", string $jargon = "", string $encoding = "", int $mode = 0): int|false {}
 
-function pspell_new_personal(string $personal, string $language, string $spelling = "", string $jargon = "", string $encoding = "", int $mode = 0): int|false {}
+function pspell_new_personal(
+    string $personal_filename,
+    string $language,
+    string $spelling = "",
+    string $jargon = "",
+    string $encoding = "",
+    int $mode = 0
+): int|false {}
 
-function pspell_new_config(int $config): int|false {}
+function pspell_new_config(int $config_id): int|false {}
 
-function pspell_check(int $pspell, string $word): bool {}
+function pspell_check(int $dictionary_id, string $word): bool {}
```

Comment:
since the manual describes the return value of `pspell_new()` as `Returns the dictionary link identifier on success or FALSE on failure.`, I think it makes sense to rename `$pspell` to `$directory_id` (the manual refers to it as `$directory_link`).

Reply:
O M G. Wow, this is even worse than usual! This extension uses resources, but instead of actually using them it works on resource IDs???

Reply:
So, considering that we should migrate these resource IDs to objects in the future, we should name them with that in mind. And we did use the `$mysql`, `$ldap` etc convention before, so keeping this at `$pspell` would make sense to me. `$dictionary_id` is sensible now, but not if we ever migrate this to a non-integer.

Reply:
I had a suspicion for a second that these are really resource IDs, but I thought that it's impossible, so rejected the idea.. Crazy!

Reply:
With that in mind, `$pspell` and`$config` seems the best choices for me as well.